quickstart/src/main/java/com/microsoft/azure/kusto/quickstart/KustoSampleApp.java

```diff
@@ -0,0 +1,399 @@
+                e.printStackTrace();
+                throw e;
+            }
+        }
+        System.out.println();
+
+        System.out.printf("Step %s: Get post-ingestion row count for '%s.%s'%n", step++, databaseName, tableName);
+        execute(kustoClient, databaseName, String.format("%s | count", tableName), ExecutionType.QUERY);
+        System.out.println();
+
+        System.out.printf("Step %s: Get sample of post-ingestion data:%n", step++);
+        execute(kustoClient, databaseName, String.format("%s | take 2", tableName), ExecutionType.QUERY);
+        System.out.println();
+    }
+
+    private static void die(String error) {
+        die(error, null);
+    }
+
+    private static void die(String error, Exception ex) {
+        System.out.println("Script failed with error: " + error);
+        if (ex != null) {
+            System.out.println("Exception:");
+            ex.printStackTrace();
+        }
+        System.exit(-1);
+    }
+
+    private static void loadConfigs() {
+        File configFile = new File(".\\" + KustoSampleApp.CONFIG_FILE_NAME);
+        Map<String, Object> configs = null;
+        try {
+            configs = new ObjectMapper().readValue(configFile, HashMap.class);
+
+            tableName = (String) configs.get("tableName");
+            databaseName = (String) configs.get("databaseName");
+            kustoUrl = (String) configs.get("kustoUri");
+            ingestUrl = (String) configs.get("ingestUri");
+            dataToIngest = (List<Map<String, String>>) configs.get("data");
+            if (StringUtils.isBlank(tableName) || StringUtils.isBlank(databaseName) || StringUtils.isBlank(kustoUrl)
+                    || StringUtils.isBlank(ingestUrl) || StringUtils.isBlank((String) configs.get("useExistingTable"))
+                    || dataToIngest.isEmpty()) {
+                die(String.format("%s is missing required fields", CONFIG_FILE_NAME));
+            }
+            useExistingTable = Boolean.parseBoolean((String) configs.get("useExistingTable"));
+            tableSchema = (String) configs.get("tableSchema"); // Optional
+        } catch (IOException e) {
+            die("Couldn't read config file", e);
+        }
+    }
+
+    private static void waitForUserToProceed(String upcomingOperation) {
+        System.out.println();
+        System.out.printf("Step %s: %s%n", step++, upcomingOperation);
+        if (WAIT_FOR_USER) {
+            System.out.println("Press ENTER to proceed with this operation...");
+            Scanner scanner = new Scanner(System.in);
+            scanner.nextLine();
+        }
+    }
+
+    private static ConnectionStringBuilder generateConnectionString(String endpointUrl, String authenticationMode) {
+        ConnectionStringBuilder csb = null;
+        switch (authenticationMode) {
+            case "UserPrompt":
+                csb = ConnectionStringBuilder.createWithUserPrompt(endpointUrl);
+                break;
+            case "ManagedIdentity":

```

Comment:
This feature is new, and is not yet merged (see https://github.com/Azure/azure-kusto-java/pull/194)